src/components/NavigationBar/index.js

```diff
@@ -2,6 +2,8 @@ import {useDispatch, useSelector} from "react-redux";
 import {isLoggedIn} from "../../redux/selectors";
```

Comment:
One more thing, can you make some changes in the App.js? Change the return value to 
```
      <>
          <NavigationBar/>
          <div className={"container"}>
              <Outlet/>
          </div>
      </>
```

Reply:
I don't understand this change. What do I have to do here?

Reply:
Changed the code to the example you have provided above.